odxtools/audience.py

```diff
@@ -19,6 +19,31 @@ class Audience:
     _enabled_audiences: Optional[List["Audience"]] = None
     _disabled_audiences: Optional[List["Audience"]] = None
 
+    @staticmethod
+    def from_et(et_element, doc_frags: List[OdxDocFragment]) \
+            -> "Audience":
+
+        enabled_audience_refs = [OdxLinkRef.from_et(ref, doc_frags)
+            for ref in et_element.iterfind("ENABLED-AUDIENCE-REFS/"
+                                           "ENABLED-AUDIENCE-REF")]
+        disabled_audience_refs = [OdxLinkRef.from_et(ref, doc_frags)
+            for ref in et_element.iterfind("DISABLED-AUDIENCE-REFS/"
+                                           "DISABLED-AUDIENCE-REF")]
+        is_supplier = et_element.get("IS-SUPPLIER", "true") == 'true'
+        is_development = et_element.get("IS-DEVELOPMENT", "true") == 'true'
+        is_manufacturing = et_element.get("IS-MANUFACTURING", "true") == 'true'
+        is_aftersales = et_element.get("IS-AFTERSALES", "true") == 'true'
+        is_aftermarket = et_element.get("IS-AFTERMARKET", "true") == 'true'
```

Comment:
Sure default value for all of those is true?

Reply:
yes:
![Screenshot_20221220_215656](https://user-images.githubusercontent.com/1837735/208764878-064b94bd-59a4-44ba-affb-3ffc3fddd77b.png)

the problem is that these attributes are also optional, so if we want to be able to distinguish "not specified", from "default" we have to interpret `None` as `True`. I've got a patch in the works for this...